django_emarsys/test_helper.py

```diff
@@ -63,3 +64,19 @@ def assertEmarsysEvents(self, events, all=True):
 
     def resetEmarsysEvents(self):
         EventInstance.objects.all().delete()
+
+
+class MockLoggingHandler(logging.Handler):
+    """
+    Mock logging handler to check for expected logs.
+
+    """
+    def __init__(self, *args, **kwargs):
+        self.reset()
+        logging.Handler.__init__(self, *args, **kwargs)
+
+    def emit(self, record):
+        self.messages.append(record)
+
+    def reset(self):
+        self.messages = []
```

Comment:
Can replace this by setting `util.log = Mock()` in the test, which then collects all method calls itself.